app/assets/javascripts/collections/user.js

```diff
@@ -0,0 +1,14 @@
+
+(function(App) {
+
+  'use strict';
+
+  App.Collection.User = App.Collection.Table.extend({
+
+    // url: 'admin/users.json'
+
+    url: '/users.json'
```

Comment:
can't we use the db sample data + API instead? JS fixtures are the source of all evils